### PR TITLE
[OC-1808] Add cypress for archive screen

### PR DESCRIPTION
**Pull request by vlo-rte**
Release notes : 

In Tasks section : 

OC-1808 : Add cypress tests for archives screen

### CODE REVIEW

File: src/test/cypress/cypress/integration/Archives.spec.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        cy.loadTestConf();
+        cy.deleteTestCards();
+        cy.deleteAllArchivedCards();
+        cy.sendTestCards();
+    });
+
+    after('Clean', function () {
+        cy.deleteTestCards();
+        cy.deleteAllArchivedCards();
+    });
```
Comment (freddidierRTE) [inline src/test/cypress/cypress/integration/Archives.spec.js:14]: Remove this clean as it is clean on test startup

File: src/test/cypress/cypress/integration/Archives.spec.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+
+describe ('Archives screen tests',function () {
+
+    before('Set up configuration', function () {
+        cy.loadTestConf();
+        cy.deleteTestCards();
```
Comment (freddidierRTE) [inline src/test/cypress/cypress/integration/Archives.spec.js:6]: Not needed on this test

**Review by freddidierRTE — CHANGES_REQUESTED** [review]

File: src/test/cypress/cypress/integration/Archives.spec.js (hunk 2 above)
Comment (vlo-rte) [reply src/test/cypress/cypress/integration/Archives.spec.js:6]: Done.

File: src/test/cypress/cypress/integration/Archives.spec.js (hunk 1 above)
Comment (vlo-rte) [reply src/test/cypress/cypress/integration/Archives.spec.js]: Done.

**Pull request merged**